modules/ROOT/nav.adoc

```diff
@@ -385,6 +385,19 @@
 ** xref:tinymce-and-cors.adoc[Cross-Origin Resource Sharing (CORS)]
 * Release information
 ** xref:release-notes.adoc[Release notes for TinyMCE 6]
+*** TinyMCE 6.1
+**** xref:6.1-release-notes.adoc#overview[Overview]
+**** xref:6.1-release-notes.adoc#accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
+**** xref:6.1-release-notes.adoc#accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
+**** xref:6.1-release-notes.adoc#bug-fixes[Bug fixes]
+**** xref:6.1-release-notes.adoc#changes[Changes]
+**** xref:6.1-release-notes.adoc#improvements[Improvements]
+**** xref:6.1-release-notes.adoc#additions[Additions]
+**** xref:6.1-release-notes.adoc#known-issues[Known issues]
```

Comment:
```suggestion
**** xref:6.1-release-notes.adoc#improvements[Improvements]
**** xref:6.1-release-notes.adoc#additions[Additions]
**** xref:6.1-release-notes.adoc#changes[Changes]
**** xref:6.1-release-notes.adoc#bug-fixes[Bug fixes]
**** xref:6.1-release-notes.adoc#known-issues[Known issues]
```
This was missed when changing the order by the looks?

Reply:
🤦